keyboards/sol/keymaps/default/keymap.c

```diff
@@ -0,0 +1,370 @@
+      _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_RGHT, KC_INS, KC_END, \
+      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, RGB_MOD, _______, _______, _______, _______, _______,     KC_MPLY, KC_MNXT, KC_MUTE, KC_VOLD, KC_VOLU, \
+                        KC_VOLU, KC_VOLD,          _______, _______, _______, _______,           KC_VOLU, KC_VOLD \
+      ),
+
+  /* ADJ
+   * ,------------------------------------------------.  ,------------------------------------------------.
+   * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |      |  |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+   * |------+------+------+------+------+------|------|  |------|------+------+------+------+------+------|
+   * |      | SAD  | VAI  | SAI  | RESET|      |      |  |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|------|  |------|------+------+------+------+------+------|
+   * |      | HUD  | VAD  | HUI  |RGBRST|      |      |  |      |      |QWERTY|COLEMK|      |      |      |
+   * |------+------+------+------+------+------+------|  |------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |  |      |      |      |RGBTOG|  HUI |  SAI | VAI  |
+   * |------+------+------+------+------+------+------|  |------+------+------+------+------+------+------|
+   * |      |      |      |RGBMOD(|      |      |      |  |      |      |      |RGBSMOD| HUD |  SAD | VAD  |
+   * `------+------+------+------+------+------+------|  |------+------+------+------+------+------+------'
+   *                                    |      |      |  |      |      |
+   *                                    `-------------'  `-------------'
+   */
+
+  [_ADJ] =  LAYOUT( \
+      KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   _______, _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12, \
+      _______, RGB_SAD, RGB_VAI, RGB_SAI, RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+      _______, RGB_HUD, RGB_VAD, RGB_HUI, RGBRST,  _______, _______, _______, _______, QWERTY,  COLEMAK, _______, _______, _______, \
+      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, \
+      _______, _______, _______, RGB_MOD, _______, _______, _______, _______, _______, _______, RGB_SMOD,RGB_HUD, RGB_SAD, RGB_VAD, \
+                        KC_VOLU, KC_VOLD,          _______, _______, _______, _______,          KC_VOLU, KC_VOLD \
+      )
+};
+
+
+
+// define variables for reactive RGB
+bool TOG_STATUS = false;
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer